app/controllers/users_controller.rb

```diff
@@ -3,17 +3,23 @@
 class UsersController < ApplicationController
   layout 'auth', only: %i[new create]
 
+  before_action :redirect_to_home, only: %i[new create]
+
   def new
-    @user = User.new
+    render locals: {
+      user: User.new
+    }
   end
 
   def create
-    @user = User.new(user_params)
+    user = User.new(user_params)
```

Comment:
I think we usually name it using action, e.g `create_params`, is that right @andyduong1920 ?
the good side is we don't need to rename the method if we have more actions require a seperate method to get the params, e.g. the `new` action may require `new_params` and so on.

Reply:
Could you please explain more? I still don't get it
Did you mean rename the method name from `user_params` to `create_params`?

Reply:
yes, I meant that - `create_params` instead of `user_params`. As explained, it indicates these params bind to the `create` action, and if we need a method to fetch the params for the `update` action, we can also name it `update_params` ...

Reply:
Fixed in 961cbe1a78ec9ce797722b8167c4ec6ad067fd73